Sources/Commands/SwiftRunCommand.swift

```diff
@@ -124,10 +124,12 @@ public struct SwiftRunCommand: AsyncSwiftCommand {
         switch options.mode {
         case .repl:
             // Load a custom package graph which has a special product for REPL.
-            let graphLoader = {
-                try swiftCommandState.loadPackageGraph(
-                    explicitProduct: self.options.executable
-                )
+            let asyncUnsafeGraphLoader = {
+                try unsafe_await {
```

Comment:
IIUC `unsafe_await` behaves the same way that `temp_await` does, it blocks the current thread? If so, what are we gaining from this change here?

Reply:
This PR lifts the unsafe ness to the highest possible point in the call stack so the benefits are:
1- The unsafe behavior is now explicitly labeled unsafe
2- Multiple unsafe blocking waits are replaced with a single one
In a future PR I make the graph loader async and remove this unsafe_await, but this seemed like a good intermediate spot on the journey to remove all of the unsafe_await style behavior. I also actually have a future PR which will make this async and reduce the usage of unsafe_await to a single call that happens off of the structured concurrency